examples/PTCDA_Hartree/run_gpu_easy.py

```diff
@@ -5,7 +5,7 @@
 from subprocess import run
 
 sys.path.append('../..')
-from pyProbeParticle.AFMulatorOCL_Simple import quick_afm
+from pyProbeParticle.ocl.AFMulatorOCL import quick_afm
```

Comment:
"easy" means without Hartree potential (with point charges) right ?

Reply:
I called it easy, because it uses this `quick_afm` convenience function that I made so that you can quickly get some AFM images just by giving an input file and some physical parameters without needing to care about the OCL stuff. It works both with point charges and Hartree.